src/CMakeLists.txt

```diff
@@ -126,23 +135,12 @@ endif(NOT BUILD_TYPE)
 
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
-# find BOOST and boost-python
-find_package( Boost COMPONENTS python REQUIRED)
-if(Boost_FOUND)
-  include_directories(${Boost_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
-  message(STATUS "found Boost: " ${Boost_LIB_VERSION})
-  message(STATUS "boost-incude dirs are: " ${Boost_INCLUDE_DIRS})
-  message(STATUS "boost-python lib is: " ${Boost_PYTHON_LIBRARY})
-  message(STATUS "boost_LIBRARY_DIRS is: " ${Boost_LIBRARY_DIRS})
-  message(STATUS "Boost_LIBRARIES is: " ${Boost_LIBRARIES})    
-endif()
-
 if(USE_OPENMP)
   if(APPLE)
     # OpenMP does not work with the default Apple llvm/clang++
     # Install a more recent version using Homebrew:
     # "brew install llvm"
-    set(OpenMP_CXX_FLAGS "-fopenmp=libomp")
+    # set(OpenMP_CXX_FLAGS "-fopenmp=libomp")
```

Comment:
I had to comment this out to make it work on macOS, we should probably try to enable it again and use a newer llvm compiler (I have tried this but probably did something wrong)